### DISCUSSION TITLE
CTkInputDialog Options

### DISCUSSION THREAD

**Discussion by BiasedGenesis** [TomSchimansky/CustomTkinter#1761] [Ideas]
Hello,

I've been working with Ctkinter to build a first app, I'm not very familiar with the underlying Tkinter library itself, so forgive me if this  question is too obtuse, but would it be possible to add extra functionalities in the dialog box? Could we make it fit it's own grid to place all of the same widgets from CTkinter within the dialog box? Or would that be something that needs to be built from scratch completely ignoring the dialog box?

**Comment by Akascape** [comment]
Use tkinter toplevel to build a customised input dialog. Please go through the documentation.

**Comment by avalon60** [comment]
> Hello,
> 
> I've been working with Ctkinter to build a first app, I'm not very familiar with the underlying Tkinter library itself, so forgive me if this question is too obtuse, but would it be possible to add extra functionalities in the dialog box? Could we make it fit it's own grid to place all of the same widgets from CTkinter within the dialog box? Or would that be something that needs to be built from scratch completely ignoring the dialog box?

Built from scratch. The input dialog, is what it is.
